dxr/plugins/python/analysis.py

```diff
@@ -100,26 +100,46 @@ class method).
             for overridden_function in overridden_functions:
                 self.overridden_functions[overridden_function].append(function)
 
-    def get_base_classes(self, absolute_class_name):
+    def get_base_classes(self, absolute_class_name, seen):
         """Return a list of all the classes that the given class
         inherits from in their canonical form.
 
+        :arg seen: The set of normalized base class names already known.  Python
+        doesn't permit actual inheritance cycles, but we don't currently
```

Comment:
Just indent the lines under ":arg seen:" for consistency with the Sphinx docs elsewhere in the project. (Also, do similarly below.)